src/nnet3/nnet-nnet.cc

```diff
@@ -455,6 +455,10 @@ int32 Nnet::GetNodeIndex(const std::string &node_name) const {
   return -1;
 }
 
+void Nnet::RenameNode(int32 node_index, const std::string &new_node_name) {
```

Comment:
I think this function should be called SetNodeName for consistency with GetNodeName(), and
document that this can be used, for example, for renaming output nodes.
But you should do some more checking here.
Do you require that after this renaming, the nnet should still be valid?  I would expect so.
 If so, then call Check() at the end of this function.  You should probably also verify that `IsValidName(new_node_name)`.